tests/cypress/integration/Home/commerceHomePage.cy.js

```diff
@@ -46,7 +46,7 @@ describe("Commerce Home Page- Coming soon mode", () => {
       .should("exist");
     cy.get("@sitePreviewFlex")
       .trigger("mouseover")
-      .find("a.nfd-button")
+      .find("a.nfd-button:first-child")
```

Comment:
@circlecube Shouldn't we do this here?

```suggestion
      .find('[data-cy="view-site"]')
```

Reply:
Yes, we can do that now. I added the data attribute as an example after updating the test.